verl/utils/torch_functional.py

```diff
@@ -162,7 +162,9 @@ def entropy_from_logits_with_chunking(logits: torch.Tensor, chunk_size: int = 20
 
 def masked_sum(values, mask, axis=None):
     """Compute mean of tensor with a masked values."""
-    return (values * mask).sum(axis=axis)
+    # If NaNs exist out of mask, replace NaNs in values with a value that won't affect the sum (e.g., 0 for masked regions)
```

Comment:
Thank you for your contribution! Could you add a test to show that it actually works with Nan?

Reply:
done~